package.json

```diff
@@ -48,7 +56,8 @@
     "lint-js": "standard && cd spec && standard",
     "lint-cpp": "python ./script/cpplint.py",
     "lint-py": "python ./script/pylint.py",
-    "lint-docs": "npm run lint-js-in-markdown && npm run create-typescript-definitions",
+    "lint-docs": "npm run lint-docs-prettier && npm run lint-js-in-markdown -- --fix && npm run create-typescript-definitions",
+    "lint-docs-prettier": "prettier --ignore-path .gitignore --write \"docs/**/*.md\" \"*.md\"",
```

Comment:
This is a bit of a nitpick -- but since prettier does more "formatting" than actual "linting", it can be confusing to call it as such, especially when we have other linting procedures in place already.